content/pages/index.rst

```diff
@@ -8,16 +8,77 @@ home
 .. raw:: html
 
     <h2 class="text-2xl text-center mb-5">Upcoming Events</h2>
-    <div class="flex flex-row flex-wrap text-center items-center justify-center">
-        <div class="md:w-1/2 w-full flex flex-row flex-wrap justify-center space-y-1">
-            <h3 class="text-lg">sciwork 2025 March scisprint in NTU Physics</h3>
-            <div class="flex-col space-x-5">
-                <a href="/sprint/2025/03-taipei.html">scisprint page</a>
-                &nbsp;&nbsp;
-            </div>
-        </div>
+        <div id="events-container" class="space-y-2">
     </div>
+     <script src="https://cdn.tailwindcss.com"></script>
+  <script>
+    const events = [      
+        { date: 'March 15, 2025 09:30:00', title: 'Taipei', type: 'Sprint', startTime: '09:30 AM', endTime: '05:00 PM', location: 'Department of Physics (National Taiwan University)', url: '/sprint/2025/03-taipei.html' },
```

Comment:
The title would be better if aligned with https://sciwork.dev/sprint/.
For example, `Taipei` should be changed to `scisprint 2025 March in Taipei`.